presto-bigquery/src/main/java/com/facebook/presto/plugin/bigquery/BigQueryClient.java

```diff
@@ -62,6 +63,52 @@
         this.viewMaterializationDataset = requireNonNull(config.getViewMaterializationDataset(), "viewMaterializationDataset is null");
     }
 
+    public BigQueryTable createTable(ConnectorTableMetadata meta)
```

Comment:
The best practice in this case is to pass in the table info with table definition and invoke Bigquery `create()` to physically create a table. Usually `CreateTableTask` involves metadata to create table with definitions passed down from `ConnectorMetadata` so the proper way to create table for Bigquery is to implement `createTable` method in BigQueryMetadata